sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/resources/application.yml

```diff
@@ -5,9 +5,14 @@ azure:
         on-demand: true
         scopes: https://management.core.windows.net/user_impersonation
       graph:
-        scopes: https://graph.microsoft.com/Calendars.Read
+        scopes:
+            - https://graph.microsoft.com/Calendars.Read
```

Comment:
Please set according to this image:
![image](https://user-images.githubusercontent.com/13167207/103045214-945a9800-45be-11eb-9497-01b2c166968a.png)